blockstore_caboose.go

```diff
@@ -67,9 +67,9 @@ func newCabooseBlockStore(orchestrator, loggingEndpoint string, cdns *cachedDNS)
 		Transport: otelhttp.NewTransport(&customTransport{
 			RoundTripper: &http.Transport{
 				// Increasing concurrency defaults from http.DefaultTransport
-				MaxIdleConns:        1000,
-				MaxConnsPerHost:     100,
-				MaxIdleConnsPerHost: 100,
+				MaxIdleConns:        200000,
+				MaxConnsPerHost:     20000,
```

Comment:
@aarshkshah1992 it is very likely removing sane limit here AND switching to HTTP1 will increase latency because you eat more cake than you can digest now, no?

Reply:
@lidel We have seen TTLB go up. What would you suggest as next steps here ?